### PR TITLE
ext

### PR DESCRIPTION
**Pull request by MrBrain295**

### CODE REVIEW

**Comment by MrBrain295** [comment]
then you can just rename it from zip to xpi and it works on firefox

**Comment by MrBrain295** [comment]
nevermind firefox dosent work

**Comment by MrBrain295** [comment]
ill work on that

**Comment by truekendor** [comment]
@MrBrain295 Many thanks, I don't understand much about workflows
So this will automatically build .xpi inside firefox dir ?

**Comment by MrBrain295** [comment]
well it dosent work as well as i thought. i will have to make some changes, but no it generates a extention file but nothing is automatic it is just something you can run on your computer

**Comment by MrBrain295** [comment]
ChatGPT made me a script that when run makes an extention that wokes for chrome (i installed chromium and tested it). I know firefox cant have the files in a directory in the archive but I cant figure out how to not do that. Here is the script `#!/bin/bash

# Define the directory to zip
SOURCE_DIR="./src"

# Define the output zip file
ZIP_FILE="extension.zip"

# Change directory to the source directory
cd "$SOURCE_DIR" || exit

# Zip all files except .ts files in the source directory
find . -type f ! -name "*.ts" -exec zip -u9 "../$ZIP_FILE" "{}" \;

# Go back to the original directory
cd ..

echo "Extension zipped successfully!"
`

**Comment by MrBrain295** [comment]
that formatted poorly

**Comment by MrBrain295** [comment]
[ext.txt](https://github.com/truekendor/better-ccc-extension/files/14971612/ext.txt)

File: package.json [hunk 1]
```diff
@@ -9,5 +9,6 @@
     "@typescript-eslint/eslint-plugin": "^6.18.1",
     "@typescript-eslint/parser": "^6.18.1",
     "eslint": "^8.56.0"
```
Comment (truekendor) [inline package.json:11]: Missing comma at the end of the line 11

**Review by truekendor — CHANGES_REQUESTED** [review]
Missing comma at the end of the line 11

**Comment by MrBrain295** [comment]
Eh I don’t think this is gonna work

**Pull request closed without merging**